ui/src/variables/components/VariablesTab.tsx

```diff
@@ -153,7 +152,7 @@ class VariablesTab extends PureComponent<Props, State> {
     router.push(`/orgs/${orgID}/settings/variables/new`)
   }
 
-  private handleUpdateVariable = (variable: Partial<Variable>): void => {
+  private handleUpdateVariable = (variable: Variable): void => {
```

Comment:
So, it would not normally work to stronger type something that was more loosely typed- but it worked because this function is not being used anymore! Could you check that that's right and remove all of this if so?

Reply:
Very astute observation!